stack/stack.go

```diff
@@ -0,0 +1,189 @@
+package stack
+
+import (
+	"errors"
+	"github.com/Appliscale/perun/context"
+	"github.com/Appliscale/perun/utilities"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/go-ini/ini"
+	"io/ioutil"
+	"os/user"
+	"time"
+)
+
+const dateFormat = "2006-01-02 15:04:05 MST"
```

Comment:
Please export this as one constant across whole code base (e.g. by creating a new module and referring it here and in `onlinevalidator.go `).

Reply:
Do you think about new .go file (package) which would have `dateFormat` only?

Reply:
My bad, I didn't notice you've merged all _cloudformation_ functions in one module. :man_facepalming: Ignore this one please.